test/CMakeLists.txt

```diff
@@ -58,14 +62,14 @@ endif()
 enable_testing()
 
 # Note: doctest and similar testing frameworks can automatically configure CMake tests For other
-# testing frameworks add the tests target instead: ADD_TEST(GreeterTests GreeterTests)
-
-include(${doctest_SOURCE_DIR}/scripts/cmake/doctest.cmake)
-doctest_discover_tests(GreeterTests)
+# testing frameworks add the tests target instead:
+add_test(greeterTests GreeterTests)
+# include(${doctest_SOURCE_DIR}/scripts/cmake/doctest.cmake) doctest_discover_tests(GreeterTests)
+# Warning: if doctest is imported with find_package() this will fail! CK
 
 # ---- code coverage ----
 
-if(ENABLE_TEST_COVERAGE)
+if(ENABLE_TEST_COVERAGE AND NOT TEST_INSTALLED_VERSION)
```

Comment:
The `ENABLE_TEST_COVERAGE` flag only makes sense when building from source, so the check is unnecessary.

Reply:
I see a problem, if you `ENABLE_TEST_COVERAGE` and install (export) this `PUBLIC` compiler options!

Reply:
As above, this is why we don't install from the tests project.